roles/podman/templates/container-.service.j2

```diff
@@ -41,5 +41,6 @@ ExecStopPost = /usr/bin/podman rm --ignore -f --cidfile %t/%n-cid
 # Downloading packages should likely be done at a different place
 # or as a ExecStartPre with higher limits on CPUQuota and MemoryMax.
 
-#CPUQuota = {{ container.cpu_quota }}
-#MemoryMax = {{ container.mem_max }}
+{% for key, value in (service | default({}) or {}).items() -%}
+{{ key }}={{ value }}
+{% endfor -%}
```

Comment:
While a good idea for flexibility, it does pushes the complexity up the stack. Leading to a more fragmented configuration for all the roles that rely on podman. Where it makes it harder to spot mis-configured services, or force specific requirements on all podman services (for example that every service would need to define a value for CPUQuota or MemoryMax limits).

Reply:
I don't see why we can't have both.

Reply:
Im against it because of how much complexity it adds to the interface we are creating. And if we run without arbitrary key=value configuration, it makes it much more desirable to put the change you want in the podman role directly, rather than each podman role having to have several arbitrary key value settings copy-pasted and updated across the repo.